code/datums/mapgen/JungleGenerator.dm

```diff
@@ -87,5 +87,5 @@
 
 /area/mine/planetgeneration
 	name = "planet generation area"
-	dynamic_lighting = DYNAMIC_LIGHTING_DISABLED
+	static_lighting = FALSE
```

Comment:
why did not having DYNAMIC lighting in the old system transition into not having STATIC lighting in the new system?

Reply:
Before it was
DYNAMIC_LIGHTING_DISABLED = always lit
DYNAMIC_LIGHTING_ENABLED = genertes lighting objects unless the area is an area that does not use power
DYNAMIC_LIGHTING_FORCED = genertes lighting objects
DYNAMIC_LIGHTING_IFSTARLIGHT genertes lighting objects depending on if the config is enabled

This has been Separated into
static_lighting : whether lighting objects get generated
base_lighting_alpha+color: how lit the area always is (ie current behavior is white at 255 alpha)
starlight lighting flag which sets static_lighting in Initialize and is checked in other starlight checks